ReactCommon/react/renderer/uimanager/primitives.h

```diff
@@ -30,13 +30,21 @@ struct ShadowNodeWrapper : public jsi::HostObject {
   ShadowNodeWrapper(SharedShadowNode shadowNode)
       : shadowNode(std::move(shadowNode)) {}
 
+  // The below method needs to be out-of-line in order for the class to have
+  // at least one "key function" (see https://itanium-cxx-abi.github.io/cxx-abi/abi.html#vague-vtable)
+  virtual ~ShadowNodeWrapper();
```

Comment:
This should use override, which implies virtual.

Reply:
Yup that's also one of the reason the internal build is failing. Thanks for pointing this out 👍